src/meta_schedule/feature_extractor/per_block_feature.cc

```diff
@@ -574,7 +1111,6 @@ class PerBlockFeatureCollector : private StmtVisitor {
   int64_t arith_intensity_curve_num_samples_;
   arith::Analyzer analyzer_;
   std::vector<const BlockRealizeNode*> scopes_;
-  std::vector<const BlockRealizeNode*> ordered_blocks_;
```

Comment:
Perhaps it's time to bring back `ordered_blocks_`.

Some context: this field is used to sort the `n` feature vectors we've got (note that they are of shape `[n, 164]` in per-store-feature) in pre-dfs-order, so that they are deterministic and easier to check

Reply:
Addressed :)